Lib/fontbakery/specifications/googlefonts.py

```diff
@@ -167,6 +167,7 @@
       , 'com.google.fonts/check/aat' # Are there unwanted Apple tables?
       , 'com.google.fonts/check/ftxvalidator_is_available' # Is the command "ftxvalidator" (Apple Font Tool Suite) available?
       , 'com.adobe.fonts/check/postscript_name_cff_vs_name' # CFF table FontName must match name table ID 6 (PostScript name).
+      , 'com.adobe.fonts/check/name_id_6_consistency' # Name table ID 6 (PostScript name) must be consistent across platforms.
```

Comment:
Could it be called **com.adobe.fonts/check/postscript_name_consistency** instead?

Reply:
Yes. Done. Thanks.